src/services/api.js

```diff
@@ -781,14 +787,16 @@ class SDKWrapper {
         const { response: { status, data } } = error;
 
         if (status === 400) return true;
-        if (status === 403 && data.message === 'Location not supported') return false;
+        if (status === 403 && data.message === t('error.locationNotSupported')) return false;
```

Comment:
It's an error message returned from the backend, so it shouldn't be checked against a translated string.

Reply:
! True, updated it. Thanks for spotting it!